pkg/acsengine/defaults.go

```diff
@@ -178,6 +178,21 @@ var (
 			},
 		},
 	}
+
+	// DefaultReschedulerAddonsConfig is the default rescheduler Kubernetes addon Config
+	DefaultReschedulerAddonsConfig = api.KubernetesAddon{
+		Name:    DefaultReschedulerAddonName,
+		Enabled: pointerToBool(true),
```

Comment:
Should this be enabled by default?

Reply:
Let's ship this as false to be cautious, and then figure out how we want to bake the feature before turning default to true.

Reply:
Also, you could pay back some of my tech debt by changing these boolean literals inside `pointerToBool()` to `api.DefaultTillerAddonEnabled` and `api.DefaultDashboardAddonEnabled` :)